libqtile/backend/wayland/wlrq.py

```diff
@@ -136,7 +137,7 @@ def paint(self, screen: Screen, image_path: str, mode: str | None = None) -> Non
         try:
             with open(image_path, "rb") as f:
                 image, _ = cairocffi.pixbuf.decode_to_image_surface(f.read())
-        except IOError:
+        except OSError:
```

Comment:
I'm confused about both of these, seems like they're both wrong?
```
>>> open("/tmp/foo/bar")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/foo/bar'
>>> isinstance(FileNotFoundError, IOError)
False
>>> isinstance(FileNotFoundError, OSError)
False
```

Reply:
```py
>>> issubclass(FileNotFoundError, IOError)
True
>>> issubclass(FileNotFoundError, OSError)
True
```

Reply:
they're not wrong, but catching a `FileNotFoundError` would probably be better either way

Reply:
Derp :), thank you! Not firing on all cylinders today apparently.